android/build.gradle

```
@@ -33,10 +33,32 @@ def getExtOrDefault(name) {
 def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['PagerView_' + name]).toInteger()
 }
+def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
+def shouldUseNameSpace = agpVersion >= 7
```

Namespace support was added with Android Gradle Plugin 7.3, [source](https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#namespace-dsl)

This comparison would break builds with AGP version `7.2`, and so on.
So I would suggest to ignore version 7 and only add the namespace property on version 8.
```suggestion
def shouldUseNameSpace = agpVersion >= 8
```